chapters/cmdbuffers.txt

```diff
@@ -106,25 +107,33 @@ on what state a command buffer must: be in, which are detailed in the valid
 usage constraints for that command.
 
 Resetting a command buffer is an operation that discards any previously
-recorded commands and puts a command buffer in the initial state.
+recorded commands and puts a command buffer in the _initial state_.
 Resetting occurs as a result of flink:vkResetCommandBuffer or
 flink:vkResetCommandPool, or as part of flink:vkBeginCommandBuffer (which
-additionally puts the command buffer in the recording state).
+additionally puts the command buffer in the _recording state_).
 
 <<commandbuffers-secondary, Secondary command buffers>> can: be recorded to
 a primary command buffer via flink:vkCmdExecuteCommands.
 This partially ties the lifecycle of the two command buffers together - if
 the primary is submitted to a queue, both the primary and any secondaries
-recorded to it move to the pending state.
-Once execution of the primary completes, so does any secondary recorded
-within it, and once all executions of each command buffer complete, they
-move to the executable state.
-If a secondary moves to any other state whilst it is recorded to another
-command buffer, the primary moves to the invalid state.
-A primary moving to any other state does not affect the state of the
-secondary.
-Resetting or freeing a primary command buffer removes the linkage to any
-secondary command buffers that were recorded to it.
+recorded to it move to the _pending state_.
+Once execution of the primary completes, so does for any secondary recorded
```

Comment:
nit: remove "for"

Reply:
Not sure I can. Execution completes **for** any secondary.

Reply:
I opted for "so it does for any". good?

Reply:
yep, that works.